src/translations/src/cboard.json

```diff
@@ -91,6 +91,15 @@
   "cboard.components.Board.emptyVoiceAlert": "WARNING: we did not detect an available Text to Speech voice! Cboard cannot work properly.",
   "cboard.components.Board.myBoardTitle": "My Board",
   "cboard.components.Board.failedToCopy": "Failed to copy to clipboard",
+  "cboard.components.Board.walkthroughWelcome": "Welcome to Cboard!",
+  "cboard.components.Board.walkthroughUnlock": "Press the lock 4 times to unlock your options and settings!",
+  "cboard.components.Board.walkthroughStart": "Are you ready for a ride across Cboard app and its awesome features?",
```

Comment:
"Are you ready for a ride across Cboard app and its awesome features?"

Should be:

"Are you ready for a tour across Cboard app and its awesome features?"

Reply:
I modified the message!